requirements/edx-sandbox/py38.in

```diff
@@ -19,4 +20,5 @@ numpy>=1.16.0,<1.17.0
 # Install these packages from the edx-platform working tree
 # NOTE: if you change code in these packages, you MUST change the version
 # number in its setup.py or the code WILL NOT be installed during deploy.
--e common/lib/sandbox-packages
```

Comment:
Since you've removed `-e common/lib/sandbox-packages`, the comment above it doesn't apply to anything.